.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: ci
+
+on:
+    pull_request:
+        branches:
+            - main
+    push:
+        branches:
+            - main
+
+jobs:
+    unit-tests:
```

```suggestion
    test:
```
We're just running the test script, which just runs `tsc`. No unit testing.